### ISSUE TITLE
Early Warning System Using BNGCNNATT

### ISSUE THREAD

**Issue by chippen001** [chippen001/Capstone-Resources#11]
### Title
An Early Warning System for Earthquake Prediction from Seismic Data Using Batch Normalized Graph Convolutional Neural Network with Attention Mechanism (BNGCNNATT)

### URL
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC9460498/ 

### Summary 
This article explains how earthquakes can destroy lives, homes, and infrastructure so developing early warning systems can provide location, magnitude, and depth to help minimize damages. CNNs (convolutional neural networks) create a feature map while GNNs (graph neural networks) uses spatial information. GCNNs can combine these to more accurately predict seismic event information. A batch normalization layer allows for more stable and faster trading of the model. This model was shown to outperform baseline models. 

### Key Points 

- Deep learning models use hidden layers: convolutional, activation, pooling, batch normalization, and attention. These extract important and useful features.
- Parameter tuning of deep learning models takes a lot of time and effort. Batch normalization trains more reliably and quickly. Enhances learning and shortens training.
- GNNs analyze data from networks: spatial data and meta-data about base stations. Effectively analyze spatial information from stations and their interactions.
- CNNs extract local features from input data and pooling learns global features. Enhances feature map and identifies valuable components. 
- Data gathered from multiple seismic stations.
- CNNs better for single station, GCNNs better for multiple station datasets
- GCNNs combine spatial information of GNN with feature map of CNN.
- In this study, events have been collected from multiple stations and station location information is included in both of the analyzed datasets.
- GCNN outperformed other baseline models with significant scores of RMSE (root mean squared error).

### Citation
Muhammad Atif Bilal, Yanju Ji, Yongzhi Wang, Muhammad Pervez Akhter, and Muhammad Yaqub. An early warning system for earthquake prediction from seismic data using batch normalized graph convolutional neural network with attention mechanism (gngcnnatt). Sensors (Basel, Switzerland), 22(17), 2022.
